components/ActionNetworkForm.vue

```diff
@@ -506,45 +512,45 @@ export default {
 
     async submitForm() {
       if (this.isSending) return
+
       this.isSending = true
 
       if (this.isJointPetition) {
-        if (!this.isUnitedStates && this.zipCode===null) {
-          this.zipCode='00000'
-        }
-        // set input vals
-        setValue('form-first_name', this.firstName)
-        setValue('form-last_name', this.lastName)
-        setValue('form-email', this.email)
-        setValue('form-street', this.address)
-        setValue('form-zip_code', this.zipCode)
-        setValue('form-comments', this.comment)
-
-        // set country val
-        setSelectValue('form-country', this.country)
-
-        // set optin
-        const els = document.querySelectorAll('input[name="subscription[group]"]')
-        for(let i=0; i < els.length; i++) {
-          els[i].checked = !this.optedOut
-        }
-
-        document.querySelector('#new_signature input[name="commit"]').click()
-      } else {
-        await this.submitToMothership()
-        this.showAfterAction()
+        this.submitJointPetitionForm()
+      }
+      else {
+        this.submitToMothership()
```

Comment:
Letting submitToMothership handle showing the after action so it can decide not to if there's an error.